config/routes.rb

```diff
@@ -82,6 +82,13 @@
           end
         end
 
+        resources :ansible_playbooks do
+          collection do
```

Comment:
Rails generate routes for all restful actions by default:

![playbook-routes](https://user-images.githubusercontent.com/2664090/135469650-9421f889-1b24-40cd-9a1b-c4f084198bc4.png)

Since we will not be using them, we do not want them to be created. We can use `:only` for that, see block below.